.github/workflows/graalvm.yml

```diff
@@ -17,12 +17,12 @@ jobs:
       - name: Git checkout
         uses: actions/checkout@v1
       - name: Set up GraalVM
-        uses: DeLaGuardo/setup-graalvm@4.0
+        uses: graalvm/setup-graalvm@v1
         with:
-          graalvm: ${{ matrix.graalvm }}
-          java: ${{ matrix.java }}
-      - name: Install native-image plugin
-        run: gu install native-image
```

Comment:
What's this `GITHUB_TOKEN`? Is it necessary, and why was it not necessary before?

Reply:
> Token for communication with the GitHub API. Please set to ${{ secrets.GITHUB_TOKEN }} (see [templates](https://github.com/bitcoinj/bitcoinj/pull/2681#templates)) to allow the action to authenticate with the GitHub API, which helps to reduce rate limiting issues.

See: https://github.com/graalvm/setup-graalvm#options

I guess the new developers are more concerned about rate-limiting than the previous developer.

Reply:
The `GITHUB_TOKEN` secret is documented here: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret